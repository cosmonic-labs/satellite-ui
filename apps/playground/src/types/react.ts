import type * as React from 'react';

declare module 'react' {
  // eslint-disable-next-line @typescript-eslint/consistent-type-definitions, @typescript-eslint/consistent-indexed-object-style, @typescript-eslint/naming-convention -- overloading
  interface CSSProperties {
    [key: `--${string}`]: string | number | undefined;
  }
}
