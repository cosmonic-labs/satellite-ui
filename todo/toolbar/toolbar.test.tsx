import {render, screen} from '@testing-library/react';
import {describe, expect} from 'vitest';
import Toolbar from './toolbar.js';

describe('Toolbar', () => {
  it('should display', () => {
    render(<Toolbar>Child Text</Toolbar>);

    expect(screen.getByTestId('toolbar')).toMatchSnapshot();
  });
});
