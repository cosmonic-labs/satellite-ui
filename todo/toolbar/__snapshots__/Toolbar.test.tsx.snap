// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Toolbar > should display 1`] = `
<div
  class="absolute left-10 top-2 z-10 flex w-20 flex-col rounded-lg bg-gray-100 text-center dark:bg-gray-700"
  data-test-id="toolbar"
>
  Child Text
</div>
`;
